src/v2/ion/uiSchemaTransformer.js

```diff
@@ -30,6 +47,55 @@ const createFactorTypeOptions = (options, factors) => {
   return options;
 };
 
+
+/**
+ * @typedef {Object} AuthenticatorInstance
+ * @property {string} authenticatorId
+ * @property {string} authenticatorType
+ * @property {string} authenticatorEnrollmentId
+ */
+/**
+ * @typedef {Object} AuthenticatorOption
+ * @property {string} label
+ * @property {Object} form
+ * @property {AuthenticatorOption[]} form.value
+ */
+/**
+ * @typedef {Object} AuthenticatorOptionValue
+ * @property {string} name
+ * @property {boolean} required
+ * @property {string} value
+ * @property {boolean} mutable
+ */
+/**
+ * Example of the option like
+ * @param {AuthenticatorOption[]} options
+ * @param {AuthenticatorInstance[]} authenticators
+ */
+const createAuthenticatorOptions = (options = [], authenticators = []) => {
+  return options.map(option => {
+    const value = option.value && option.value.form && option.value.form.value || [];
+    const valueObject = value
+      // TODO:
+      // base on current API design, there maybe field that has
```

Comment:
in which case does it come up?

Reply:
Phone's method type as current proposal.
and in theory that's one possible combination.